### PR TITLE
Added README file 

### PR DESCRIPTION
**Pull request by shreenaathia**
Let me know if there are any details you would like changed on this file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+ Python - https://www.python.org/downloads/ 
+ Text Editor  - eg Visual Studio Code (if planning to edit this repo) 
+
+
+### Installing
+
+Clone this Repo  
+
+In Command Line Prompt
```
Comment (jrkong) [inline README.md]: Terminal for OSX and Linux and Command Prompt for Windows

File: README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### Installing
+
+Clone this Repo  
+
+In Command Line Prompt 
+
+To run use 
+
+python C:\Users\pySearch\pySearch.py  ( different path name where you have cloned pySearch into)
```
Comment (jrkong) [inline README.md]: Could you also list what all the command line arguments are? Note that the `-s` argument is required as it denotes your search. 

Run the script using `-h` to see a brief description of what each argument does. You can also read the code to see what the command line arguments do.

Take at the code or try running the code and calling the `-h`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+python C:\Users\pySearch\pySearch.py  ( different path name where you have cloned pySearch into) 
+
+
+## Running the tests
+
+To be added .. 
+
+## Built With
+
+Python
```
Comment (jrkong) [inline README.md]: Python 3 to be exact. Python 2 has different syntax and some of the libraries used don't exist on Python 2.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+Software you will need
+
+ Python - https://www.python.org/downloads/
```
Comment (jrkong) [inline README.md]: Make these a list, I think it will make it easier to read.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project. (TO be added) 
+
+
+## Goals 
+
+The goal of pySearch is to bring web search capabilities to your command line so initiating a web search doesn't require taking your hand off the keyboard. The current goal is to integrate as much of the web search workflow into the command line so users can get all the keyboard interaction out of the way before taking one hand off the keyboard to use the mouse to interact with the browser.
+
+### Specific Goals 
+
+Support basic webcrawling which would allow basic searches, basic dictionary lookup would be a great test case for this, to be performed before opening a browser. (think Unix man pages)
```
Comment (jrkong) [inline README.md]: These items could be put in a list to improve readability.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+
+## Authors
+
+* **Alex Kong** - *Initial work* ( https://github.com/jrkong )
+
+See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project. (TO be added) 
+
+
+## Goals
```
Comment (jrkong) [inline README.md]: I think Objective would fit better here.

**Comment by shreenaathia** [comment]
Hey I made all the changes you requested and added some more improvements on the 4th commit

**Comment by jrkong** [comment]
Solid first start. It covers all the base functionality. Now we'll need to keep this updated as we add new features.

**Pull request merged**